conf/app.routes

```diff
@@ -195,3 +195,5 @@ POST       /registration/change-business-address            controllers.Business
 GET        /registration/business-details-not-match             controllers.BusinessDetailsDoNotMatchController.onPageLoad()
 
 GET        /registration/business-details-match-other               controllers.BusinessDetailsMatchOtherController.onPageLoad()
+
+GET        /registration/business-already-registered       controllers.BusinessAlreadyRegisteredController.onPageLoad()
```

Comment:
`controllers.BusinessAlreadyRegisteredController.onPageLoad()` should be inline with the rest. Correct indentation please :)